src/main/webapp/scripts/gallery.js

```diff
@@ -11,11 +11,11 @@ var $grid = $('.grid').masonry({
     horizontalOrder: true
 });
 
-fetchAllSpeciesData("test.json");
+fetchAllSpeciesData();
 
 // Test function that fetches sample JSON and appends each species to the gallery
 function fetchAllSpeciesData(data) {
-    fetch(data).then(response => response.json()).then(speciesData => {
+    fetch("/allData").then(response => response.json()).then(speciesData => {
```

Comment:
I think Tom suggested that we pass the string as an argument in line 14 (i.e. fetchAllSpeciesdata("/allData"))

Reply:
https://github.com/googleinterns/step129-2020/pull/18#discussion_r453442533

Reply:
I think that might just be because it was using a test json file before. I think it should be okay now?

Reply:
Ohh that makes sense!